aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -12,3 +12,83 @@ proposals:
       - Framework:
           bytecode_version: 6
           git_hash: ~
+  - name: change_operator_beneficiary
+    metadata:
+      title: "Changing beneficiaries for operators"
+      description: "AIP-51: This allows operator to change their beneficiaries."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/251"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - OPERATOR_BENEFICIARY
+  - name: change_commission_rate_delegation_pool
+    metadata:
+      title: "Change commission rates in delegation pools"
+      description: "AIP-50: This allows changing commission rates in delegation pools."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/249"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - COMMISSION_CHANGE_DELEGATION_POOL
+  - name: reduce_ident_size_max
+    metadata:
+      title: "Reduce the maximum identifier size"
+      description: "This reduces the maximum identifier size for (de)serializer of Move binary format"
+      discussion_url: "https://github.com/aptos-labs/aptos-core/pull/10502"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - LIMIT_MAX_IDENTIFIER_LENGTH
+  - name: introduce_single_sender_authenticator
+    metadata:
+      title: "Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts"
+      description: "AIP-55: Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-55.md#aip-55---generalize-transaction-authentication-and-support-arbitrary-k-of-n-multikey-accounts"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - SINGLE_SENDER_AUTHENTICATOR
+  - name: allow_optional_fee_payer
+    metadata:
+      title: "Make Fee Payer Address Optional in Transactions Authenticator"
```

Comment:
Why not put the AIP in the title as well? Here and in other entries

Reply:
Added the AIP numbers to the title as well.